### DISCUSSION TITLE
Google Sync, Easter Egg and Extension!

### DISCUSSION THREAD

**Discussion by guranotfound** [Alex313031/thorium#514] [Q&A]
What I don't understand is: How does Thorium add Google Sync, Easter Egg and uBlock features? Actually, this is my first time building Chromium, can anyone give me details? @@

**Comment by Alex313031** [comment]
@GuraNotFound Google Sync is by adding API Keys at the top of your args.gn. You can use the ./fetch_api_keys.sh script in //infra to download them.

The Easter Egg is embedded in the code. If you wanna know how to change it, let me know.

Installing uBlock Origin by default is accomplished with this file, which is copied when making an installer .exe or .deb > https://github.com/Alex313031/thorium/blob/main/infra/default_apps/external_extensions.json

**Reply by guranotfound** [reply]
Thanks so much for your help!

**Comment by guranotfound** [comment]
Thank you for your help! Now I want to know how you can incorporate photos to become an "Easter Egg" in it? I'm very grateful for that.

________________________________
Từ: Alex F. ***@***.***>
Đã gửi: 19 Tháng Giêng 2024 6:01 SA
Đến: Alex313031/thorium ***@***.***>
Cc: GuraNotFound ***@***.***>; Mention ***@***.***>
Chủ đề: Re: [Alex313031/thorium] Google Sync, Easter Egg and Extension! (Discussion #514)


@GuraNotFound<https://github.com/GuraNotFound> Google Sync is by adding API Keys at the top of your args.gn. You can use the ./fetch_api_keys.sh script in //infra to download them.

The Easter Egg is embedded in the code. If you wanna know how to change it, let me know.

Installing uBlock Origin by default is accomplished with this file, which is copied when making an installer .exe or .deb > https://github.com/Alex313031/thorium/blob/main/infra/default_apps/external_extensions.json
